README.md

```diff
@@ -63,16 +63,18 @@ To change build tags from the default, set the `BUILDTAGS` variable for make,
 e.g.
 
 ```bash
-make BUILDTAGS='seccomp apparmor'
+make BUILDTAGS='seccomp'
 ```
 
 | Build Tag | Feature                            | Enabled by default | Dependency |
 |-----------|------------------------------------|--------------------|------------|
 | seccomp   | Syscall filtering                  | yes                | libseccomp |
 | selinux   | selinux process and mount labeling | yes                | <none>     |
-| apparmor  | apparmor profile support           | yes                | <none>     |
 | nokmem    | disable kernel memory accounting   | no                 | <none>     |
 
+The following tags were used in the past versions, but no longer meaningful in the
+latest version:
+- apparmor
```

Comment:
I would be more specific here, as "latest version" is vague.

Something like 

```suggestion
The following build tags were used earlier, but are now obsoleted:
 - **apparmor** (since runc v1.0.0-rc93 the feature is always enabled)
```

(or something like that -- the main idea is to mention a specific version)

Reply:
fixed